### PR TITLE
Dependency update and some hack/test

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

Fixes #

## Proposed Changes

-
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: test/e2e-common.sh [hunk 1]
```diff
@@ -564,3 +564,10 @@ function parse_flags() {
   esac
   return 0
 }
+
+## hack
+function get_latest_knative_yaml_source() {
+  echo "https://storage.googleapis.com/knative-releases/eventing/latest/eventing.yaml"
+}
+
+readonly KNATIVE_EVENTING_RELEASE="$(get_latest_knative_yaml_source)"
```
Comment (devguyio) [inline test/e2e-common.sh:573]: I think you missed removing the condition on line 174 to always install eventing from that version?

Comment (matzew) [reply test/e2e-common.sh:573]: donks!

I indeed forgot that

Comment (devguyio) [reply test/e2e-common.sh:573]: turns out that URL resolves to nightly releases :disappointed: 
```
>> Install Knative Eventing from https://storage.googleapis.com/knative-nightly/eventing/latest/eventing.yaml
```

Comment (devguyio) [reply test/e2e-common.sh:573]: one quick fix to unblock main maybe is to git checkout latest release and use this funcion to get the latest release? https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c 
so the code in else (lines 178..) can continue but on the checked out release?

just thinking out loud

**Review by devguyio — CHANGES_REQUESTED** [review]
It still installs from nightly, left a comment with an idea

File: test/e2e-common.sh (hunk 1 above)
Comment (matzew) [reply test/e2e-common.sh:573]: let's see what the tmp URL with explicit pin-point says

**Comment by matzew** [comment]
@devguyio   if I directly apply 0.22  ... or check it out.... there is IMO no difference .

**Comment by devguyio** [comment]
@matzew yeah sure, I was mainly saying to "detect" the latest release, but I'm fine having this go in for now and fixing it properly after we're unblocked

**Review by devguyio — APPROVED** [review]
lgtm

**Pull request merged**